### ISSUE TITLE
enforce min/max in range controls input

### ISSUE THREAD

**Issue by azrafe7** [fponticelli/sui#18]
While the slider is forced to move only inside the specified range, the input control accepts values outside of it.

Not sure there's a _clean_ way to solve this (as the user might have different needs, etc.), just bringing it to your attention.

You can test it by typing (f.e.) `0` as `Vertex Count` [here](https://rawgit.com/azrafe7/hxClipper/master/bin/js/index.html), which is [set to have a range 3-100](https://github.com/azrafe7/hxClipper/blob/master/src%2FSuiDemoJS.hx#L111-L115).

Maybe I'm doing it wrong?

<sub>I've tried to force it (clamping the value to be in the range) in the setter, but then I cannot write `20` f.e., because as soon as I write `2` it gets corrected to `3`.</sub>

**Comment by fponticelli** [comment]
I don't think you are doing it wrong and it is reasonable to assume that the value should always be contained within the range. I will have a look.
